stubs/nanoid/nanoid/method.pyi

```diff
@@ -0,0 +1,6 @@
+from collections.abc import Callable
+from typing_extensions import TypeAlias
+
+_Algorithm: TypeAlias = Callable[[int], bytearray | list[int]]
```

Comment:
Eyeballing the source code, could this be slightly simpler and less restrictive?

```suggestion
from collections.abc import Callable, Sequence
from typing_extensions import TypeAlias

_Algorithm: TypeAlias = Callable[[int], Sequence[int]]
```

Reply:
nice, that's definitely better 👍